libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -838,6 +856,42 @@ void _IotMqtt_DecrementConnectionReferences( _mqttConnection_t * pMqttConnection
 
 /*-----------------------------------------------------------*/
 
+/* Void Function pointer Implementation as we need to pass it in setting the MQTT Context used
```

Comment:
this is not a function pointer, but just an implementation. Please correct the comments.